adutils/src/main/java/com/appyhigh/adutils/AdSdk.kt

```diff
@@ -1177,21 +1218,26 @@ object AdSdk {
                     .build()
             )
             .build()
-        loadAd(adLoader, context, adUnit, "e")
+        loadAd(adLoader, context, adUnit, "e", contentURL, neighbourContentURL)
     }
 
 
     private fun loadAd(
         adLoader: AdLoader?,
         context: Context,
         adUnit: String,
-        s: String
+        s: String,
```

Comment:
What is this s variable?

Reply:
removed them as it was mainly used for debug logging to identify the ad units and location